src/sass/objects/_product.scss

```diff
@@ -0,0 +1,3 @@
+.product-share-cnt {
```

Comment:
If this is the only styling that this class adds, we should consider creating a helper class instead.